cmd/commands/cli/command.go

```diff
@@ -26,14 +28,28 @@ import (
 
 	"github.com/chzyer/readline"
 	"github.com/mysteriumnetwork/node/cmd"
+	"github.com/mysteriumnetwork/node/core/service"
 	"github.com/mysteriumnetwork/node/metadata"
+	"github.com/mysteriumnetwork/node/services/noop"
+	"github.com/mysteriumnetwork/node/services/openvpn"
+	openvpn_service "github.com/mysteriumnetwork/node/services/openvpn/service"
+	"github.com/mysteriumnetwork/node/services/wireguard"
+	wireguard_service "github.com/mysteriumnetwork/node/services/wireguard/service"
 	tequilapi_client "github.com/mysteriumnetwork/node/tequilapi/client"
 	"github.com/mysteriumnetwork/node/utils"
 	"github.com/urfave/cli"
 )
 
 const cliCommandName = "cli"
 
+const serviceHelp = `service <action> [args]
+	start	<ProviderID> <ServiceType> [options]
```

Comment:
`start	<ProviderID> <ServiceType> [<OpenvpnPort>] [<OpenvpnProto>]`

Reply:
But it can be any options depending on the `ServiceType`. Do you want to hardcode all allowed now flags?

Reply:
ok, clear now